### DISCUSSION TITLE
2022-50: Sccache 和 OpenDAL 的故事

### DISCUSSION THREAD

**Discussion by giscus** [Xuanwo/blog#391] [Announcements]
# 2022-50: Sccache 和 OpenDAL 的故事

An infrastructure engineer, focused on distributed storage system

https://xuanwo.io/reports/2022-50/

**Comment by beyoung** [comment]
文中最后出现了 Datbaend 可见最近是真的高兴😄

**Reply by Endle** [reply]
趁着作者还没改过来，赶快提交到 Internet Archive

**Reply by Xuanwo** [reply]
感谢指出，已经修复～

**Comment by Arcovv** [comment]
真的很厲害！

**Comment by xxchan** [comment]
我觉得 OpenDAL 比起先辈能成功的一大原因就是它真的有人用了（databend），然后就能滚雪球了 🤪
